src/py/aspen/database/models/sample.py

```diff
@@ -14,6 +18,9 @@
 from .base import idbase
 from .usergroup import Group
 
+if TYPE_CHECKING:
+    from .sequences import SequencingReads, UploadedPathogenGenome
```

Comment:
What happens if `TYPE_CHECKING` is not true when `SequencingReads` is used on line 167?

Reply:
I'm guessing it just ignores the entire line where it's used as it looks like it's just a type specification, is that right?

Reply:
When you do `from __future__ import annotations`, it ignores type annotations when not type checking.